---
title: "Services: genomics & transcriptomics"
#toc: False
menu:
    main:
        name: Services
        identifier: genomics_transcriptomics_services
        parent: genomics_transcriptomics
        weight: 30
---

## SciLifeLab National Genomics Infrastructure

At the SciLifeLab [National Genomics Infrastructure (NGI)](https://ngisweden.scilifelab.se/),
we provide a wide range of sequencing technologies and can offer state-of-the-art solutions for
many different types of COVID-19 sequencing projects.

In addition to [instrumentation](https://ngisweden.scilifelab.se/technologies/)
and next-generation sequencing, NGI provides [library preparation](https://ngisweden.scilifelab.se/applications/)
services and support with project planning and [primary bioinformatics analysis](https://ngisweden.scilifelab.se/bioinformatics/).

Many of the sample and experiment types that we work with could be of use in COVID-19 related research,
whether studying the virus itself or its interactions with the host.

### Access to the infrastructure

The main focus of the NGI is Swedish academic research groups, though we also work with industry and international groups.

Please visit our website [https://ngisweden.scilifelab.se/contact/](https://ngisweden.scilifelab.se/contact/)
to find out more about our services, and contact us at [support@ngisweden.se](mailto:support@ngisweden.se)
to discuss your specific project.

## Rapid cDNA and direct RNA sequencing of SARS-CoV-2 using Oxford Nanopore

RNA viruses (including SARS-CoV-2, MERS, SARS) continually accumulate changes in their genomes
that can be used to reconstruct the epidemiological processes that drive the spread of the virus.
Most countries with an active outbreak have sequenced viruses from multiple individuals, allowing
tracking of viral clades and geographic origin of regional outbreaks.

The NGI has been granted money from [SciLifeLab](https://www.scilifelab.se/covid-19) and the
[Knut & Wallenberg Foundation (KAW)](https://kaw.wallenberg.org/)
to set up a protocol for SARS-CoV-2 sequencing using the fast and portable
[Oxford Nanopore MinION sequencer](https://nanoporetech.com/products/minion).
With this grant, we intend to establish two different protocols to enable rapid analysis of the
SARS- CoV-2 genome based on cDNA sequencing and direct RNA sequencing.

There are multiple benefits to using nanopore sequencing for initial screening.
First, it is a rapid protocol that makes it possible to provide a consensus viral genome in just 8 hours,
and up to 24 samples can be multiplexed on a single MinION flow-cell.
Second, it is a protocol that is easily transferred and can be set up quickly within clinics or
research labs for decentralized real-time viral tracking.

By establishing these protocols at NGI we can offer the capacity to sequence samples submitted by researchers or clinicians.
Although we see larger cohort sequencing to be crucial going forward using through-put approaches,
we foresee a need to establish a protocol that can be implemented quickly before large scale screening efforts are in place.
We also expect a need for research projects to be able to submit samples to NGI for cDNA amplicon synthesis and sequencing,
e.g. for analysis of specific subgroups of patients or sampling from different geographic regions.

### ARTIC sequencing protocol

We intend to set up the already established and validated protocol available through the
[ARTIC network](https://artic.network/ncov-2019).
The ARTIC network is an organization focused on tracking evolution and spread of viral epidemics, making all sequencing protocols open source and enabling data sharing.
There is a validated protocol available for SARS-CoV-2 sequencing, including primers, laboratory protocols, bioinformatics tutorials and datasets.
An important aspect of the ARTIC network is also the sharing of data across the world.
The protocol has been widely used in a number of countries including China, UK and Iceland.

### Direct RNA sequencing

It is possible to sequence RNA directly using the Oxford Nanopore MinION, without first converting the RNA to cDNA.
We aim to exploit this unique feature to establish protocols enabling direct sequencing of the entire
RNA genome of SARS-CoV-2.
This makes it possible to not only determine the RNA bases of the viral genome,
but also epigenetic RNA base modifications, and if desirable, the host cell RNA repertoire.
Methylation of specific nucleotides in the SARS-CoV-2 genomes have been reported,
but their potential role in virus biology, virus replication or host-pathogen interaction
are unknown and an important area to investigate further.

For more information about the development and availability of these techniques, please
get in touch  at [support@ngisweden.se](mailto:support@ngisweden.se).

<!--
## Access to platform

## Data type specific support

## Guidelines for data producers

### Metadata

### Analysis

### Publishing

## Links to data type specific Data Hub services

-->
